### PR TITLE
Feature/enable swiftlint for non intelusers

### PR DESCRIPTION
**Pull request by Suraj1999**

### CODE REVIEW

File: animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift [hunk 1]
```diff
@@ -90,7 +90,11 @@ final class FeedingPointDetailsModel: FeedingPointDetailsModelProtocol, FeedingP
         let history = try await context.feedingPointsService.fetchFeedingHistory(for: fullFeedingPoint.identifier)
         guard !history.isEmpty else { return [] }
 
-        let sortedByDateHistory = history.sorted(by: { $0.updatedAt > $1.updatedAt })
+//        let sortedByDateHistory = history.sorted(by: { $0.updatedAt > $1.updatedAt })
```
Comment (kishorepran) [inline animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift]: Please remove this commented code.

File: animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift [hunk 2]
```diff
@@ -90,7 +90,11 @@ final class FeedingPointDetailsModel: FeedingPointDetailsModelProtocol, FeedingP
...
+        let sortedByDateHistory = history.sorted { item1, item2 in
```
Comment (kishorepran) [inline animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift]: please check for higher order short hand syntax here and try.
https://www.appcoda.com/higher-order-functions-swift/#Sorted

**Review by raghvendrapacholiepam — APPROVED** [review]

File: animeal/src/Flows/Profile/ProfileViewController.swift [hunk 3]
```diff
@@ -67,7 +67,8 @@ final class ProfileViewController: BaseViewController, ProfileViewable {
         navigationItem.hidesBackButton = viewConfiguration.hidesBackButton
         if !viewConfiguration.hidesBackButton {
             navigationItem.hidesBackButton = true
-            let backButton = UIBarButtonItem(image: UIImage(named: Asset.Images.arrowBackOffset.name), style: .plain, target: self, action: #selector(backTapped))
+            let img = UIImage(named: Asset.Images.arrowBackOffset.name)
```
Comment (kishorepran) [inline animeal/src/Flows/Profile/ProfileViewController.swift]: nit: variable names should be meaningful

File: animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift (hunk 1 above)
Comment (Suraj1999) [reply animeal/src/Flows/Main/Modules/Home/FeedingPointDetails/Model/FeedingPointDetailsModel.swift]: Done

File: animeal/src/Flows/Profile/ProfileViewController.swift (hunk 3 above)
Comment (Suraj1999) [reply animeal/src/Flows/Profile/ProfileViewController.swift]: Done

**Review by kishorepran — APPROVED** [review]
LGTM

**Pull request merged**